src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java

```diff
@@ -0,0 +1,23 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalImport"/>
+  </module>
+</module>
+*/
+
+//non-compiled with javac: Compilable with Java8
+package com.puppycrawl.tools.checkstyle.checks.imports.illegalimport;
+
+// xdoc section -- start
+import java.io.*;
+import java.lang.ArithmeticException;
+import java.sql.Connection;
+import java.util.List;
+import java.util.Enumeration;
+import java.util.Arrays;
+import java.util.Date;
+import sun.applet.*; // violation, illegal import of all sun packages by default
```

Comment:
Please put in single quotes real violation part.

Example https://github.com/checkstyle/checkstyle/commit/9db7149daa9466798d8915f70527de1780be06ec#diff-c8eb9255bd0ecac76e703a44339630ceabf20d576c9a6db4985bd204fbd6072dR16

Please same for all lines

Reply:
done